Backup-manager looks for programs from the cdrkit suite.
Since cdrtools are not part of Debian, backup-manager searches for
wodim instead of cdrecord and genisoimage instead of mkisofs.

Provided by: Sven Joachim <svenjoac@gmx.de>

---
 lib/externals.sh |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/lib/externals.sh
+++ b/lib/externals.sh
@@ -7,10 +7,10 @@
 dar=$(which dar 2> /dev/null) || true
 tar=$(which tar 2> /dev/null) || true
 rsync=$(which rsync 2> /dev/null) || true
-mkisofs=$(which mkisofs 2> /dev/null) || true
+mkisofs=$(which genisoimage 2> /dev/null) || true
 growisofs=$(which growisofs 2> /dev/null) || true
 dvdrwformat=$(which dvd+rw-format 2> /dev/null) || true
-cdrecord=$(which cdrecord 2> /dev/null) || true
+cdrecord=$(which wodim 2> /dev/null) || true
 md5sum=$(which md5sum 2> /dev/null) || true
 bc=$(which bc 2> /dev/null) || true
 mysqldump=$(which mysqldump 2> /dev/null) || true
